### DISCUSSION TITLE
Sentido da Vida

### DISCUSSION THREAD

**Discussion by vish-blog** [vish-blog/comments#288] [Imagens]
Post de referência; https://www.vish.blog.br/imagens/tirinhas/sentido-da-vida/

**Comment by brunoaugusto** [comment]
![null](<https://i.postimg.cc/BQVfCs1k/sentido-da-vida.png>)

**Comment by angellisy** [comment]
> [Bruno Augusto](</perfil/?user=brunoaugusto>) disse [segunda-feira, 4 de abril de 2022](<https://www.vish.blog.br/imagens/tirinhas/sentido-da-vida/#comment_1087736439>):

> [![null](<https://camo.githubusercontent.com/ff2e393395d165b5cc6529758c8d0ca9cb6ce841a68667cca0c18bad17f8af9e/68747470733a2f2f692e706f7374696d672e63632f425156664373316b2f73656e7469646f2d64612d766964612e706e67>)](<https://camo.githubusercontent.com/ff2e393395d165b5cc6529758c8d0ca9cb6ce841a68667cca0c18bad17f8af9e/68747470733a2f2f692e706f7374696d672e63632f425156664373316b2f73656e7469646f2d64612d766964612e706e67>)

Esse sim é um verdadeiro "Momento de Sabedoria" ^^

**Comment by one-eyed-willy** [comment]
> [angellisy](</perfil/?user=angellisy>) disse [segunda-feira, 4 de abril de 2022](<https://www.vish.blog.br/imagens/tirinhas/sentido-da-vida/#comment_1087737382>):

> Esse sim é um verdadeiro "Momento de Sabedoria" ^^

Concordo, devia mandar como sugestão

**Comment by brunoaugusto** [comment]
> [Willy Caolho](</perfil/?user=one-eyed-willy>) disse [segunda-feira, 4 de abril de 2022](<https://www.vish.blog.br/imagens/tirinhas/sentido-da-vida/#comment_1087738263>):

> Concordo, devia mandar como sugestão

Eu amo O Incrível Mundo de Gumball, mas é tão difícil achar esse *frame* específico com boa qualidade. Nunca vi em mais do que 720p, nem em Inglês!



Mas quem sabe, é uma ideia...
